.github/workflows/zxc-e2e-test.yaml

```diff
@@ -182,7 +182,7 @@ jobs:
         uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
         with:
           name: solo-${{ inputs.npm-test-script }}.log
-          path: ~/.solo/logs/solo.log
+          path: ~/.solo/logs/*
```

Comment:
assuming that you add the namespace as I mentioned below, then will want to retest this to make sure it uploads subdirectories and their files

Reply:
updated and tested